packages/components/src/confetti/index.ts

```diff
@@ -58,12 +58,12 @@ function fireConfetti() {
 	} );
 }
 
-const ConfettiAnimation = ( { trigger = true, delay = 0 } ) => {
+const ConfettiAnimation = ( { trigger = true, delay = 0, colors = [] } ) => {
```

Comment:
> should this default the the same empty array?

I'm not sure what you exactly mean by this. Could you please explain it a bit more?

Reply:
Every render of `ConfettiAnimation` where you don't specify the third `colors` argument will render it with a new array, because `[]` basically creates a new array. To fix that, we can create a constant `const EMPTY_ARRAY = []` above, and then use that as the default value of `colors`. Let me know if that makes more sense.

Reply:
Thanks! I didn't know that.
I have a PR that will use the `COLORS` const as a default in case there is no value provided.

https://github.com/Automattic/wp-calypso/pull/85419